controllers/chat/getSignedChannelList.js

```diff
@@ -40,7 +58,7 @@ const getSignedChannelList = async (req, res) => {
   const queriedChannels = [];
   const promisedChannels = [];
 
-  const client = GetstreamSingleton.getChatInstance();
+  const client = new StreamChat(process.env.API_KEY, process.env.API_SECRET);
   try {
     await client.connectUser({id: req.userId}, req.token);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Creating a new instance of `StreamChat` every time the `getChannels` function is called might not be efficient. If there's a reason to create a new instance each time, such as needing a separate connection per request, then it's fine. However, if that's not necessary, consider using a singleton pattern to reuse the same instance across multiple requests.



<!-- This is an auto-generated comment by OSS CodeRabbit -->